src/test/java/de/butbetter/keksys/VATCalculatorTest.java

```diff
@@ -0,0 +1,27 @@
+package de.butbetter.keksys;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class VATCalculatorTest {
+
+	private VATCalculator calc;
+
+	@BeforeEach
+	void setup() {
+		calc = new VATCalculator();
+	}
```

Comment:
test some edge cases like Float.MinValue .../ 0 0 / Float.MaxValue

Reply:
I don't know what are you mean?

Reply:
just some extra test cases with maybe some parameters like:
Float.MinValue .../ 0 0 / Float.MaxValue

Float.MaxValue and 1 MAYBE? lel

Reply:
Next one try to explain a explain with assert explain not like this .../ 0 0 /  👍  💯 
More like this:
```java
 assertEquals(0,calc.calculateVAT(0,0));
```